[2017-05-18 09:24:16.632] [ERROR] logErr - 执行sql发生错误：INSERT INTO bd_organ (organ_id,organ_name,is_root,has_child,create_time,p_organ_id,p_organ_name)VALUES('65343', '北京教育部', '1', '0', 'undefined', '2123', '北京市委') ,('65344', '北京教育部', '1', '0', 'undefined', '2124', '北京市委') ,('65345', '北京教育部', '1', '0', 'undefined', '2125', '北京市委') ,('65346', '北京教育部', '1', '0', 'undefined', '2126', '北京市委') /n 错误信息：Error: ER_TRUNCATED_WRONG_VALUE: Incorrect datetime value: 'undefined' for column 'create_time' at row 1
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Pool.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Pool.js:199:23)
    at Queries.query (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:10:9)
    at Object.queryStrSql (F:\work\QHPG-BJDF\util\mysqlUtil.js:25:16)
    at Object.saveInfo (F:\work\QHPG-BJDF\dao\bdmanage_dao\organ_coDao.js:28:20)
    at Object.saveUserInfo (F:\work\QHPG-BJDF\service\bdmanage_service\organ_coService.js:25:21)
    at F:\work\QHPG-BJDF\routes\bdmanage_route\organ_coRoute.js:11:21
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
    at next (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:110:13)
    at Route.dispatch (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:91:3)
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
[2017-05-18 09:24:16.643] [ERROR] logErr - 错误信息：Error: ER_TRUNCATED_WRONG_VALUE: Incorrect datetime value: 'undefined' for column 'create_time' at row 1
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Pool.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Pool.js:199:23)
    at Queries.query (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:10:9)
    at Object.queryStrSql (F:\work\QHPG-BJDF\util\mysqlUtil.js:25:16)
    at Object.saveInfo (F:\work\QHPG-BJDF\dao\bdmanage_dao\organ_coDao.js:28:20)
    at Object.saveUserInfo (F:\work\QHPG-BJDF\service\bdmanage_service\organ_coService.js:25:21)
    at F:\work\QHPG-BJDF\routes\bdmanage_route\organ_coRoute.js:11:21
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
    at next (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:110:13)
    at Route.dispatch (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:91:3)
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
[2017-05-18 09:24:55.351] [ERROR] logErr - 执行sql发生错误：INSERT INTO bd_organ (organ_id,organ_name,is_root,has_child,p_organ_id,p_organ_name)VALUES('65343', '北京教育部', '1', '0', 'undefined', '2123', '北京市委') ,('65344', '北京教育部', '1', '0', 'undefined', '2124', '北京市委') ,('65345', '北京教育部', '1', '0', 'undefined', '2125', '北京市委') ,('65346', '北京教育部', '1', '0', 'undefined', '2126', '北京市委') /n 错误信息：Error: ER_WRONG_VALUE_COUNT_ON_ROW: Column count doesn't match value count at row 1
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Pool.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Pool.js:199:23)
    at Queries.query (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:10:9)
    at Object.queryStrSql (F:\work\QHPG-BJDF\util\mysqlUtil.js:25:16)
    at Object.saveInfo (F:\work\QHPG-BJDF\dao\bdmanage_dao\organ_coDao.js:28:20)
    at Object.saveUserInfo (F:\work\QHPG-BJDF\service\bdmanage_service\organ_coService.js:25:21)
    at F:\work\QHPG-BJDF\routes\bdmanage_route\organ_coRoute.js:11:21
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
    at next (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:110:13)
    at Route.dispatch (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:91:3)
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
[2017-05-18 09:24:55.358] [ERROR] logErr - 错误信息：Error: ER_WRONG_VALUE_COUNT_ON_ROW: Column count doesn't match value count at row 1
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Pool.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Pool.js:199:23)
    at Queries.query (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:10:9)
    at Object.queryStrSql (F:\work\QHPG-BJDF\util\mysqlUtil.js:25:16)
    at Object.saveInfo (F:\work\QHPG-BJDF\dao\bdmanage_dao\organ_coDao.js:28:20)
    at Object.saveUserInfo (F:\work\QHPG-BJDF\service\bdmanage_service\organ_coService.js:25:21)
    at F:\work\QHPG-BJDF\routes\bdmanage_route\organ_coRoute.js:11:21
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
    at next (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:110:13)
    at Route.dispatch (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:91:3)
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
[2017-05-18 09:26:33.841] [ERROR] logErr - 执行sql发生错误：INSERT INTO bd_organ (organ_id,organ_name,is_root,has_child,p_organ_id,p_organ_name)VALUES('65343', '北京教育部', '1', '0', 'undefined', '2123', '北京市委') ,('65344', '北京教育部', '1', '0', 'undefined', '2124', '北京市委') ,('65345', '北京教育部', '1', '0', 'undefined', '2125', '北京市委') ,('65346', '北京教育部', '1', '0', 'undefined', '2126', '北京市委') /n 错误信息：Error: ER_WRONG_VALUE_COUNT_ON_ROW: Column count doesn't match value count at row 1
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Pool.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Pool.js:199:23)
    at Queries.query (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:10:9)
    at Object.queryStrSql (F:\work\QHPG-BJDF\util\mysqlUtil.js:25:16)
    at Object.saveInfo (F:\work\QHPG-BJDF\dao\bdmanage_dao\organ_coDao.js:28:20)
    at Object.saveUserInfo (F:\work\QHPG-BJDF\service\bdmanage_service\organ_coService.js:25:21)
    at F:\work\QHPG-BJDF\routes\bdmanage_route\organ_coRoute.js:11:21
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
    at next (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:110:13)
    at Route.dispatch (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:91:3)
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
[2017-05-18 09:26:33.856] [ERROR] logErr - 错误信息：Error: ER_WRONG_VALUE_COUNT_ON_ROW: Column count doesn't match value count at row 1
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Pool.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Pool.js:199:23)
    at Queries.query (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:10:9)
    at Object.queryStrSql (F:\work\QHPG-BJDF\util\mysqlUtil.js:25:16)
    at Object.saveInfo (F:\work\QHPG-BJDF\dao\bdmanage_dao\organ_coDao.js:28:20)
    at Object.saveUserInfo (F:\work\QHPG-BJDF\service\bdmanage_service\organ_coService.js:25:21)
    at F:\work\QHPG-BJDF\routes\bdmanage_route\organ_coRoute.js:11:21
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
    at next (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:110:13)
    at Route.dispatch (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:91:3)
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
[2017-05-18 09:29:27.990] [ERROR] logErr - 执行sql发生错误：INSERT INTO bd_organ (organ_id,organ_name,is_root,has_child,p_organ_id,p_organ_name)VALUES('65343', '北京教育部', '1', '0', 'undefined', '2123', '北京市委') ,('65344', '北京教育部', '1', '0', 'undefined', '2124', '北京市委') ,('65345', '北京教育部', '1', '0', 'undefined', '2125', '北京市委') ,('65346', '北京教育部', '1', '0', 'undefined', '2126', '北京市委') /n 错误信息：Error: ER_WRONG_VALUE_COUNT_ON_ROW: Column count doesn't match value count at row 1
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Pool.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Pool.js:199:23)
    at Queries.query (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:10:9)
    at Object.queryStrSql (F:\work\QHPG-BJDF\util\mysqlUtil.js:25:16)
    at Object.saveInfo (F:\work\QHPG-BJDF\dao\bdmanage_dao\organ_coDao.js:28:20)
    at Object.saveUserInfo (F:\work\QHPG-BJDF\service\bdmanage_service\organ_coService.js:25:21)
    at F:\work\QHPG-BJDF\routes\bdmanage_route\organ_coRoute.js:11:21
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
    at next (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:110:13)
    at Route.dispatch (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:91:3)
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
[2017-05-18 09:29:27.999] [ERROR] logErr - 错误信息：Error: ER_WRONG_VALUE_COUNT_ON_ROW: Column count doesn't match value count at row 1
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Pool.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Pool.js:199:23)
    at Queries.query (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:10:9)
    at Object.queryStrSql (F:\work\QHPG-BJDF\util\mysqlUtil.js:25:16)
    at Object.saveInfo (F:\work\QHPG-BJDF\dao\bdmanage_dao\organ_coDao.js:28:20)
    at Object.saveUserInfo (F:\work\QHPG-BJDF\service\bdmanage_service\organ_coService.js:25:21)
    at F:\work\QHPG-BJDF\routes\bdmanage_route\organ_coRoute.js:11:21
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
    at next (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:110:13)
    at Route.dispatch (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:91:3)
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
[2017-05-18 09:41:58.122] [ERROR] logErr - 执行sql发生错误：UPDATE bd_gradeinfo SET grade_name='6年级',xueduan='2' WHERE grade_id='2';, WHERE grade_id='3';,UPDATE bd_gradeinfo SET grade_name='8年级' WHERE grade_id='4';/n 错误信息：Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'WHERE grade_id='3'' at line 1
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Protocol._enqueue (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at PoolConnection.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:208:25)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:40:13
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:718:13
    at iterate (F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:262:13)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:274:29
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:44:16
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:723:17
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:167:37
    at Query._callback (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:42:7)
[2017-05-18 09:41:58.129] [ERROR] logErr - 错误信息：Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'WHERE grade_id='3'' at line 1
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Protocol._enqueue (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at PoolConnection.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:208:25)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:40:13
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:718:13
    at iterate (F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:262:13)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:274:29
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:44:16
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:723:17
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:167:37
    at Query._callback (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:42:7)
[2017-05-18 09:44:51.202] [ERROR] logErr - 执行sql发生错误：DELETE FROM bd_gradeinfo WHERE user_id IN (undefined,undefined,undefined)/n 错误信息：Error: ER_BAD_FIELD_ERROR: Unknown column 'user_id' in 'where clause'
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Pool.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Pool.js:199:23)
    at Queries.query (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:10:9)
    at Object.queryStrSql (F:\work\QHPG-BJDF\util\mysqlUtil.js:25:16)
    at Object.removeInfo (F:\work\QHPG-BJDF\dao\bdmanage_dao\gradeDao.js:39:20)
    at Object.removeUserInfo (F:\work\QHPG-BJDF\service\bdmanage_service\gradeService.js:45:18)
    at F:\work\QHPG-BJDF\routes\bdmanage_route\gradeRoute.js:15:18
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
    at next (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:110:13)
    at Route.dispatch (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:91:3)
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
[2017-05-18 09:44:51.203] [ERROR] logErr - 错误信息：Error: ER_BAD_FIELD_ERROR: Unknown column 'user_id' in 'where clause'
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Pool.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Pool.js:199:23)
    at Queries.query (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:10:9)
    at Object.queryStrSql (F:\work\QHPG-BJDF\util\mysqlUtil.js:25:16)
    at Object.removeInfo (F:\work\QHPG-BJDF\dao\bdmanage_dao\gradeDao.js:39:20)
    at Object.removeUserInfo (F:\work\QHPG-BJDF\service\bdmanage_service\gradeService.js:45:18)
    at F:\work\QHPG-BJDF\routes\bdmanage_route\gradeRoute.js:15:18
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
    at next (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:110:13)
    at Route.dispatch (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:91:3)
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
[2017-05-18 09:49:14.850] [ERROR] logErr - 执行sql发生错误：DELETE FROM bd_gradeinfo WHERE grade_id IN (undefined,undefined,undefined)/n 错误信息：Error: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'where clause'
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Pool.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Pool.js:199:23)
    at Queries.query (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:10:9)
    at Object.queryStrSql (F:\work\QHPG-BJDF\util\mysqlUtil.js:25:16)
    at Object.removeInfo (F:\work\QHPG-BJDF\dao\bdmanage_dao\gradeDao.js:39:20)
    at Object.removeUserInfo (F:\work\QHPG-BJDF\service\bdmanage_service\gradeService.js:45:18)
    at F:\work\QHPG-BJDF\routes\bdmanage_route\gradeRoute.js:15:18
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
    at next (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:110:13)
    at Route.dispatch (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:91:3)
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
[2017-05-18 09:49:14.860] [ERROR] logErr - 错误信息：Error: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'where clause'
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Pool.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Pool.js:199:23)
    at Queries.query (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:10:9)
    at Object.queryStrSql (F:\work\QHPG-BJDF\util\mysqlUtil.js:25:16)
    at Object.removeInfo (F:\work\QHPG-BJDF\dao\bdmanage_dao\gradeDao.js:39:20)
    at Object.removeUserInfo (F:\work\QHPG-BJDF\service\bdmanage_service\gradeService.js:45:18)
    at F:\work\QHPG-BJDF\routes\bdmanage_route\gradeRoute.js:15:18
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
    at next (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:110:13)
    at Route.dispatch (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:91:3)
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
[2017-05-18 09:49:46.518] [ERROR] logErr - 执行sql发生错误：DELETE FROM bd_gradeinfo WHERE grade_id IN (undefined,undefined,undefined)/n 错误信息：Error: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'where clause'
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Pool.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Pool.js:199:23)
    at Queries.query (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:10:9)
    at Object.queryStrSql (F:\work\QHPG-BJDF\util\mysqlUtil.js:25:16)
    at Object.removeInfo (F:\work\QHPG-BJDF\dao\bdmanage_dao\gradeDao.js:39:20)
    at Object.removeUserInfo (F:\work\QHPG-BJDF\service\bdmanage_service\gradeService.js:45:18)
    at F:\work\QHPG-BJDF\routes\bdmanage_route\gradeRoute.js:15:18
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
    at next (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:110:13)
    at Route.dispatch (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:91:3)
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
[2017-05-18 09:49:46.530] [ERROR] logErr - 错误信息：Error: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'where clause'
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Pool.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Pool.js:199:23)
    at Queries.query (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:10:9)
    at Object.queryStrSql (F:\work\QHPG-BJDF\util\mysqlUtil.js:25:16)
    at Object.removeInfo (F:\work\QHPG-BJDF\dao\bdmanage_dao\gradeDao.js:39:20)
    at Object.removeUserInfo (F:\work\QHPG-BJDF\service\bdmanage_service\gradeService.js:45:18)
    at F:\work\QHPG-BJDF\routes\bdmanage_route\gradeRoute.js:15:18
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
    at next (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:110:13)
    at Route.dispatch (F:\work\QHPG-BJDF\node_modules\express\lib\router\route.js:91:3)
    at Layer.handle [as handle_request] (F:\work\QHPG-BJDF\node_modules\express\lib\router\layer.js:82:5)
